source/common/http/conn_manager_utility.cc

```diff
@@ -171,6 +171,14 @@ ConnectionManagerUtility::MutateRequestHeadersResult ConnectionManagerUtility::m
       if (result.reject_options.has_value()) {
         return {nullptr, result.reject_options};
       }
+      if (result.append_xff) {
+        if (Network::Utility::isLoopbackAddress(
```

Comment:
maybe worth a wrapper utility given this looks like a copy paste of preexisting code?

Reply:
I've factored this code out to a wrapper function.